js/private/js_image_layer.bzl

```diff
@@ -18,9 +18,19 @@ load("@bazel_skylib//lib:paths.bzl", "paths")
 
 _DOC = """Create container image layers from js_binary targets.
 
-js_image_layer supports transitioning to specific platform for cross-compiling.
+By design, js_image_layer doesn't have any preference over rule which assembles the container image. 
+However, this comes with a few quirks; the downstream rule, `oci_image` or `container_image` in this case,
+has to set proper `workdir` and `cmd` to make the container work.
```

Comment:
Should we clarify what a "proper" workdir is? I'm not sure what "proper" means in this case...

Reply:
that's what is done down below. starts with "Usually"